asterisk/config.yaml

```diff
@@ -15,15 +15,27 @@ map:
   - config:rw
   - ssl
 options:
+  ami_password: ''
   video_support: true
   auto_add: true
+  auto_add_secret: ''
   certfile: fullchain.pem
   keyfile: privkey.pem
+  mailbox: false
+  mailbox_port: 12345
+  mailbox_password: ''
+  mailbox_extension: '101'
+  api_key: GOOGLE_API_KEY
```

Comment:
All default passwords should be removed, so that the user is forced from the Supervisor UI to add one (if mandatory).

https://github.com/TECH7Fox/Asterisk-add-on/pull/73

If it's optional (like auto_add_secret when auto_add is disabled), check how I did in the PR.

Reply:
How about this?

```
options:
  video_support: true
  auto_add: true
  certfile: fullchain.pem
  keyfile: privkey.pem
schema:
  ami_password: password
  video_support: bool
  auto_add: bool
  auto_add_secret: password?
  certfile: str
  keyfile: str
  mailbox: bool
  mailbox_port: port?
  mailbox_password: password?
  mailbox_extension: str?
  api_key: password?
```

Reply:
looks nice!

Some notes though:

- I think mailbox should be false by default
- I think mailbox_port should have a default value, even though it may not be used when mailbox: false
- I think api_key could be named mailbox_google_api_key instead
- I think mailbox_extension could also use a default value

Reply:
Should we add the port setting to ports? Works with `host_network`, it just changes that port, and then we can set a description too. Like the motioneye add-on. (Which also uses host_network)
![image](https://user-images.githubusercontent.com/32220029/152251602-2fe02445-98ef-4d75-ad8f-2630373f7fc8.png)

Reply:
I think it may give the wrong impression to the user that they can tweak/disable the ports. When using `host_network`, nothing about this matters. Except the description and visibility, which is helpful of course.

To address the latter, the ports can be added to the DOCS.md. What do you think?

Reply:
Really? Thought it did work. Then we should do that yes.

Did the other things you mentioned. Should be good now.